### PR TITLE
replace links with button on sign up and sign in

### PR DESCRIPTION
**Pull request by amal-san**
#29 links change to buttons

### CODE REVIEW

File: frontend/src/Header.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ import React from 'react'
 import { useStore } from './hooks'
 import authService from './services/auth'
 
+const authbutton = {
```
Comment (germanescobar) [inline frontend/src/Header.js:5]: Can you move these styles to App.scss?

Comment (amal-san) [reply frontend/src/Header.js]: Sure :+1:

Comment (amal-san) [reply frontend/src/Header.js]: :+1:

**Comment by amal-san** [comment]
#21 also add fix for map height

**Comment by germanescobar** [comment]
@amal-san why did you close the PR? I was going to tell you that there are some conflicts that we need to fix before merging. Let me know if you need help with this.

**Comment by germanescobar** [comment]
@amal-san thanks for the PR! We had some internal changes of files and we included these changes there, so I will close this PR for now. I would suggest you fork the project again to have the latest changes. Thanks!

**Pull request closed without merging**